commands/info.md

```diff
@@ -433,6 +440,13 @@ For each database, the following line is added:
 
 *   `dbXXX`: `keys=XXX,expires=XXX`
 
+The **debug** section is for some experimental or temporary metrics. It will not be displayed when `INFO` or even `INFO ALL` is called, and it is only displayed when `INFO DEBUG` is called.
```

Comment:
I'm not certain if we wanna explicitly document this section, or just leave it undocumented.
i suppose documenting (as you did), it with an explicit warning is better.. @madolson WDYT?
```suggestion
The **debug** section is for experimental metrics, which might change or get removed in future versions. It will not be displayed when `INFO` or even `INFO ALL` is called, and it is only displayed when `INFO DEBUG` is called.
```

Reply:
I would slightly prefer not documenting it, just like we don't document debug configs in redis.conf. If someone cares enough they can look at the code. It's not a strong preference though, I'm fine adding a warning that the fields may disappear at any time.